WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -314,3 +315,50 @@ $venue-map-wrapper-height: 400px;
     background-color: inherit;
   }
 }
+
+table.admin-competitions {
+  font-size: 13px;
+
+  .hide-tbody {
+    display: none;
+  }
```

Comment:
Actually we don't need this class, Bootstrap provides such, see [here](http://getbootstrap.com/css/#helper-classes-show-hide).